src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -180,10 +195,8 @@
 						<Setter Target="PaneColumnGrid.CornerRadius" Value="0,8,0,8" />
 						<Setter Target="ContentPresenter.(Grid.Column)" Value="0" />
 						<Setter Target="ContentPresenter.(Grid.ColumnSpan)" Value="3" />
-						<Setter Target="Content.CornerRadius" Value="0" />
-						<Setter Target="Content.BorderThickness" Value="0,1,0,1" />
 						<Setter Target="SidebarResizer.Visibility" Value="Collapsed" />
-						<Setter Target="ContentPresenter.Margin" Value="0,0,0,0" />
+						<Setter Target="ContentPresenter.Margin" Value="8,0,8,0" />
```

Comment:
The regular margin is `2,0,8,0` but in the narrow view the sidebar is hidden and thus, we change to `8,0,8,0`.